flexmeasures_client.py

```diff
@@ -299,28 +299,21 @@ def trigger_schedule(self, *args, **fnc_kwargs):
 
         if schedule_id is None:
             self.log_failed_response(res, url)
-            self.handle_response_errors(message, res, url, self.trigger_schedule, *args, **fnc_kwargs)
+            self.try_solve_authentication_error(message, res, url, self.trigger_schedule, *args, **fnc_kwargs)
             self.set_state("input_boolean.error_schedule_cannot_be_retrieved", state="on")
             return None
 
         self.log(f"Successfully triggered schedule. Schedule id: {schedule_id}")
         self.set_state("input_boolean.error_schedule_cannot_be_retrieved", state="off")
         return schedule_id
 
-    def handle_response_errors(self, message, res, description, fnc, **fnc_kwargs):
+    def try_solve_authentication_error(self, message, res, description, fnc, **fnc_kwargs):
         if fnc_kwargs.get("retry_auth_once", True) and res.status_code == 401:
             self.log(f"Failed to {description} on authorization (possibly the token expired); attempting to "
                      f"reauthenticate once.")
             self.authenticate_with_fm()
             fnc_kwargs["retry_auth_once"] = False
             fnc(**fnc_kwargs)
```

Comment:
`*fnc_args` should be passed back to the function, too.

Reply:
Done